src/Umbraco.Core/Security/MembershipProviderBase.cs

```diff
@@ -249,6 +268,7 @@ public override void Initialize(string name, NameValueCollection config)
             _maxInvalidPasswordAttempts = GetIntValue(config, "maxInvalidPasswordAttempts", 5, false, 0);
             _passwordAttemptWindow = GetIntValue(config, "passwordAttemptWindow", 10, false, 0);
             _minRequiredPasswordLength = GetIntValue(config, "minRequiredPasswordLength", DefaultMinPasswordLength, true, 0x80);
+            _maxRequiredPasswordLength = 256;
```

Comment:
Could the default value here come from a constant? It's reused in a few places so would be safer